generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -7024,7 +7277,8 @@ namespace nidaqmx_grpc {
       return ::grpc::Status::CANCELLED;
     }
     try {
-      auto session_name = request->session_name().c_str();
+      auto session_name_mbcs = convert_from_grpc<std::string>(request->session_name());
+      auto session_name = session_name_mbcs.c_str();
 
       auto init_lambda = [&] () {
```

Comment:
Ok, this is correctly passing the MBCS session name to DAQmx. The session repository uses the UTF-8 session name, which seems fine because the session lookup code also uses the UTF-8 session name.